opcua/server/server.py

```diff
@@ -318,12 +318,12 @@ def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
 
-    def get_event_object(self, etype=ua.ObjectIds.BaseEventType, source=ua.ObjectIds.Server):
+    def get_event_object(self, event=ua.BaseEvent()):
```

Comment:
and that it should take etype=ua.ObjectIds.BaseEventType as argument. As we discussed earlier, a events are something dynamic that can be generated on the fly from the standard namespace or whatever homemade structure